prboom2/src/r_bsp.c

```diff
@@ -40,6 +40,7 @@
 #include "r_bsp.h" // cph - sanity checking
 #include "v_video.h"
 #include "lprintf.h"
+#include "p_spec.h"
```

Comment:
Do we still need this here?

Reply:
No, and I thought of the same later on too. :)